scripts/update_python_version.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# This script is use to update the version of ITK used by ITKPythonPackage. It
+# is run nightly by a system with push permissions to the
+#
+#   git@github.com:InsightSoftwareConsortium/ITKPythonPackage.git
+#
+# repository.
+
+set -x -e
+
+workdir=/tmp/
+
+cd $workdir
+
+if test ! -d ITKNightlyMaster; then
+  git clone --branch nightly-master \
+    https://github.com/InsightSoftwareConsortium/ITK.git ITKNightlyMaster
+fi
+pushd ITKNightlyMaster/
+git checkout nightly-master
+git pull
+popd
+
+if test ! -d ITKPythonPackage; then
+  git clone --branch master \
+    git@github.com:InsightSoftwareConsortium/ITKPythonPackage.git ITKPythonPackage
+fi
+pushd ITKPythonPackage/
+git checkout master
+git pull
+./scripts/update_python_version.py ../ITKNightlyMaster
+git add -- CMakeLists.txt itkVersion.py
+git commit -m "ITK nightly-master version update" --author='Kitware Robot <kwrobot@kitware.com>'
```

Comment:
I would suggest something more explicit:

```
ITKPythonPackage version update

This commit updates:
  (1) SHA used in CMakeLists.txt to checkout ITK sources (InsightSoftwareConsortium/ITK@${ITK_SHA})
  (2) VERSION variable set in itkVersion.py

It was automatically generated by the script ``update_python_version.sh`` [1]

[1] https://github.com/InsightSoftwareConsortium/ITKPythonPackage/blob/master/scripts/update_python_version.py
```

Variable ITK_SHA would be set in the `pushd ITKNightlyMaster/` section above using:

```
ITK_SHA=$(git rev-parse --short HEAD)
```